crm_lead_date_duration/views/crm_lead_views.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_crm_lead_form_inherit" model="ir.ui.view">
+        <field name="name">crm_lead_form (in CRM Lead Date Duration)</field>
+        <field name="model">crm.lead</field>
+        <field name="inherit_id" ref="crm.crm_lead_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//group//field[@name='date_deadline']" position="before">
+                <field name="event_date" />
+                <label for="duration" attrs="{'invisible': [('duration','=',False)]}"/>
+                <div attrs="{'invisible': [('duration','=', 0)]}">
```

Comment:
This invisible condition makes duration impossible to be edited, could you check?